lib/core/janitor.js

```diff
@@ -50,18 +50,18 @@ class Janitor {
    * @param {object} securities
    * @returns {Promise}
    */
-  async loadSecurities (securities = {}, userId = '-1') {
+  async loadSecurities (securities = {}, user = null) {
```

Comment:
The method `admin:loadSecurities` should forward the current user to this method

Reply:
It's already done https://github.com/kuzzleio/kuzzle/pull/1563/files#diff-8be4ba2ed5b9aab3d1b100866dcc323aR176